geomesa-convert/geomesa-convert-common/src/main/scala/org/locationtech/geomesa/convert/CompositeConverter.scala

```diff
@@ -42,17 +42,22 @@ class CompositeConverter[I](val targetSFT: SimpleFeatureType,
                                converters: Seq[(Predicate, SimpleFeatureConverter[I])])
   extends SimpleFeatureConverter[I] {
 
-  override def processInput(is: Iterator[I]): Iterator[SimpleFeature] =
+  override def processInput(is: Iterator[I],  gParams: Option[Map[String, String]] = None): Iterator[SimpleFeature] = {
+    if (gParams.isDefined) ec.globalParams = gParams
```

Comment:
The execution context has a map of variable to value - it's stored as an array and the indexes are precomputed for performance during lookup.  We should append the global params to the array for fast lookup as well.